test/createFunction/createFunction.Script.v2.test.ts

```diff
@@ -146,6 +146,16 @@ function addSuite(tester: FunctionTesterBase): void {
             inputs: [],
             skip: tester.language === ProjectLanguage.Custom
         },
+        {
+            functionName: 'Durable Functions entity',
+            inputs: [],
+            skip: tester.language === ProjectLanguage.PowerShell || tester.language === ProjectLanguage.Java
+        },
+        {
+            functionName: 'Durable Functions Entity HTTP starter',
+            inputs: tester.language === ProjectLanguage.JavaScript ? [] : [getRotatingAuthLevel()],
+            skip: tester.language === ProjectLanguage.PowerShell || tester.language === ProjectLanguage.Java || tester.language === ProjectLanguage.Python
```

Comment:
Does this not exist for Python? I'm wondering why you decided to skip testing this one.

Reply:
Yes, it doesn't exist yet. I'll be adding it later, apologies for the inconsistency across languages. We did not notice this was missing, in part, because the templates don't show up by default :)